roles/contiv_network/tasks/main.yml

```diff
@@ -108,3 +108,6 @@
 
 - include: aci_tasks.yml
   when: (run_as == "master") and (contiv_network_mode == "aci")
+
+- include: fwd_tasks.yml
+  when: (run_as == "master") 
```

Comment:
no this is a global setting. Doesnt need to be run on all nodes. It just needs to be run on master.